src/backend/api/meals.js

```diff
@@ -2,14 +2,94 @@ import express from "express";
 const router = express.Router();
 import knex from "../database.js";
 
-router.get("/", async (request, response) => {
+// router.get("/", async (request, response) => {
+//   try {
+//     // knex syntax for selecting things. Look up the documentation for knex for further info
+//     const titles = await knex("meal").select("title");
+//     response.json(titles);
+//   } catch (error) {
+//     throw error;
+//   }
+// });
+
+// maxPrice Number  Returns all meals that are cheaper than maxPrice. api/meals?maxPrice=90
+// availableReservations  Boolean Returns all meals that still have available spots left, if true. If false, return meals that have no available spots left.1 api/meals?availableReservations=true
+//  title String  Returns all meals that partially match the given title. Rød grød will match the meal with the title Rød grød med fløde. api/meals?title=Indian%20platter
+// dateAfter  Date  Returns all meals where the date for when is after the given date.  api/meals?dateAfter=2022-10-01
+// dateBefore Date  Returns all meals where the date for when is before the given date. api/meals?dateBefore=2022-08-08
+// sortKey  String  Returns all meals sorted by the given key. Allows when, max_reservations and price as keys. Default sorting order is asc(ending). api/meals?sortKey=price
+// sortDir  String  Returns all meals sorted in the given direction. Only works combined with the sortKey and allows asc or desc. api/meals?sortKey=price&sortDir=desc
+// limit  Number  Returns the given number of meals.  api/meals?limit=7
```

Comment:
Is all this commented code needed? If not please remove it :)

Reply:
actually it was the assignments. I should have deleted or separated for each part🙈